docs/NOTIFICATION.md

```diff
@@ -0,0 +1,136 @@
+# Notification system
+
+With Web Chat 4.8, you can start displaying notifications that represents information that are important for your customers. Currently, we render most notifications as toasts on the top of the screen. Connectivity status also become part of the notification system.
+
+## Adding and updating notification
+
+To make both add and update operation seamless, we are using an "upsert" strategy. When you add a new notification, you will need to specify `id`. To update an existing notification, you will use the same `id` of the notification you want to update.
+
+Markdown is supported for the message. When using Markdown, you are responsible to provide a screen reader version of your message via `alt` field. Hyperlinks will have `rel="noopener noreferrer"` and `target="_blank"` added for privacy reasons.
+
+### Using Redux actions
+
+```js
+dispatch({
+  type: 'WEB_CHAT/SET_NOTIFICATION',
+  payload: {
+    alt: 'Please read our privacy policy at https://privacy.microsoft.com.'
+    id: 'your-id',
+    level: 'info',
+    message: 'Please read our [privacy policy](https://privacy.microsoft.com/).'
+  }
+});
+```
+
+### Using React hooks
+
+```js
+const setNotification = useSetNotification();
+
+setNotification({
+  alt: 'Please read our privacy policy at https://privacy.microsoft.com.'
+  id: 'your-id',
+  level: 'info',
+  message: 'Please read our [privacy policy](https://privacy.microsoft.com/).'
+});
+```
+
+## Dismissing a notification
+
+To dismiss a notification, you will need to pass the `id` to identify individual notification.
+
+### Using Redux actions
+
+```js
+dispatch({
+   type: 'WEB_CHAT/DISMISS_NOTIFICATION',
+   payload: {
+      id: 'your-id'
+   }
+});
+```
+
+### Using React hooks
+
+```js
+const setNotification = useSetNotification();
+
+setNotification({
+   id: 'your-id'
+});
+```
+
+## User experience
+
+Web Chat implements the notification UI as toast, located above the transcript. We offer multiple style options to customize the UI. In case the design and user experience is not desirable in target applications, developers are welcomed to rebuild the UI.
```

Comment:
```suggestion
Web Chat implements the notification UI as toast, which are located above the transcript. We offer multiple style options to customize the UI. In case the design and user experience are not desirable in target applications, developers are welcomed to rebuild the UI.
```

Reply:
"Web Chat implements toast notifications, which are..."
```suggestion
Web Chat implements toast notifications, which are located above the transcript. We offer multiple style options to customize the UI. In case the design and user experience is not desirable in target applications, developers are welcomed to rebuild the UI.
```

Reply:
@compulim please go through documentation and move away from plural pronouns?